### ISSUE TITLE
include Taylor diagrams:

### ISSUE THREAD

**Issue by fbaffie** [NVE/Flood_forecasting#32]
Z:\Fastgrupper\Flomvarsling\Flomvarslingssystem\hbv\MudellEvaluering\Modell_Evaluering2017.xlsm

Byman had already done the Taylor diagrams. I need to look more into what he's done but maybe he can help also. We need to discuss how we want to integrate this. Maybe rather in a New Version of the app based on Jan's ideas

**Comment by ElinLangsholt** [comment]
Yes, rather a separate app, ref. #29 (could be merged). Rel. low priority.
